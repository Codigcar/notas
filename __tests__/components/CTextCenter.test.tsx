import React from 'react';

const {render} = require('@testing-library/react-native');
const {CTextCenter} = require('../../src/components/CTextCenter');
// import renderer from 'react-test-renderer';

describe('testing en <CTextCenter />', () => {
    it('debe hacer match con el snapshot', () => {
        const container = render(<CTextCenter title='title' />)
        expect(container).toMatchSnapshot()
    });
});
