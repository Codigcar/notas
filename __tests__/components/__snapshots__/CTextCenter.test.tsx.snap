// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing en <CTextCenter /> debe hacer match con el snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View>
    <Text>
      title
    </Text>
  </View>
</View>
`;
